streamnative/README.md

```diff
@@ -0,0 +1,81 @@
+# StreamNative
+
+## Overview
+
+[StreamNative][1] provides an enterprise-grade messaging and event streaming platform built on Apache Pulsar. It offers scalable, real-time data streaming solutions with features like multi-tenancy, geo-replication, and seamless integration with cloud services.
+
+The StreamNative integration collects below types of metrics:
+
+1. Health
+2. Pulsar Resource
+3. Source Connector
+4. Sink Connector
+5. Kafka Connect
+
+## Setup
+
+### Configuration
+
+StreamNative integration requires a StreamNative account and its Client Id, Client Secret, Organization ID and Instance Name. Below are the steps to fetch these details from StreamNative console:
+
+#### Steps to retrieve Organization ID, Instance Name, Client Id & Client Secret
+
+1. Login to the [StreamNative Cloud Console Account][2] 
+2. Go to the profile icon and select to the **Organizations** option.
+3. Go into the **Organization** from which data needs to be collected in the Datadog.
+4. Copy and save the **Organization ID** to configure in the Integration.
+5. From the **Select an Instance** section, Copy and save the **Instance Name** to configure in the Integration.
+6. Go to the profile icon and select the **Accounts & Accesses** tab.
+7. Choose the Service Account for which **Admin** permission is **True**.
```

Comment:
Admin permission is True > `Admin permission is enabled`

Reply:
Done.